CIP-0044?/README.md

```diff
@@ -0,0 +1,89 @@
+
+## Motivation
+
+CNFT marketplaces struggle to keep up with verifying new collection policyID’s due to the influx of new creators and projects. This standard would relieve the market spaces from verifying new policyID’s and could instead display the social media accounts of the creator who owns the collection. From here the consumer is left to do their due diligence. This process also provides the marketplace with the creator name, collection name and possibly an Avatar\PFP if one is available from the platform hosting the URI.
+
+## Specification
+
+A new tag of 808 is proposed for this implementation.
+
+1. The creator makes a new Cardano policy.
+2. They then use one or more existing online platforms where they have a social reputation, one that their potential consumers find trustworthy. On this platform they create content that contains the policyID of the NFT collection in question. If using a Handle, the owner adds the policyID as an augmentation.
+3. The 808 verification tags are to be written to an unnamed asset, using the policy to be used for the intended Cardano Assets.
+4. Updates or rewrites are allowed.
+5. Within this created asset will be the metadata for verification. It will use a tag of 808, and then have three tags to identify the name of the collection, the name of the creator, and URI. Those tags will be “collection”, “creator”, and “URI” respectively.
+6. The “collection”, “creator” and “URI” key tags can be any UTF-8 value or array. By allowing for an array, any tag can exceed the per line 64 character limitation.
+7. All markets will be instructed to look for the latest minted 808 asset on a policy and cross reference the URI document policyID data with the 808 asset. If they match, the projectID can be considered validated. The market could then issue a rating system. If multiple URI’s were validated they could show a higher validation score. The creators social media handles could be displayed next to the verified collection or asset.
+
+## Example JSON with string
+
+```
+{
+	"808": {
```

Comment:
It would be good to specify this further with actual types and also describe how you handle the transaction metadata size limitations (ex: is "create" always just a string? Can it be an array to get over the 64 byte limit?)